src/Service.Tests/SqlTests/RestApiTests/Patch/PatchApiTestBase.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
```

Comment:
is this needed? at least in my GitHub code review view, I see only updates to a comment and whitespace.

Reply:
I don't think so, wonder how it got added.